packages/protocol-kit/src/Safe.ts

```diff
@@ -794,7 +794,7 @@ class Safe {
       operation: serviceTransactionResponse.operation,
       safeTxGas: serviceTransactionResponse.safeTxGas,
       baseGas: serviceTransactionResponse.baseGas,
-      gasPrice: Number(serviceTransactionResponse.gasPrice),
+      gasPrice: serviceTransactionResponse.gasPrice,
```

Comment:
According to https://safe-transaction-mainnet.safe.global/, `baseGas` and `safeTxGas` in lines 795-796 are integers so we should convert them to strings if that is true